src/App.scss

```diff
@@ -0,0 +1,53 @@
+
+.header {
+  width: 100%;
+  height: 300%;
+  background: url("./assets/home.jpg") bottom center no-repeat;
+  background-size: cover;
+  padding: 100px;
+
+  &-content {
+    width: 215px;
+    color: #fff;
+
+    &__title {
+      font-size: 22px;
+      font-weight: 700;
+      text-shadow: 1px 2px 2px rgba($color: #000, $alpha: 0.6);
+    }
+
+    &__desc {
+      font-weight: 100;
+      text-shadow: 0 1px 1px rgba($color: #000, $alpha: 0.4);
+    }
+  }
+}
+
+.btn-oval {
+  border-radius: 35px !important;
+  background-color: #ea531c !important;
+  border-color: #ea531c !important;
+  color: #fff !important;
+  padding: 5px 20px;
+}
+
+.btn-menu {
+  border-radius: 35px !important;
+  background-color: #787877 !important;
+  border-color: #787877 !important;
+  color: #fff !important;
+  padding: 5px 30px;
+}
+
+.auth_links {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin: 0;
+  padding: 0;
+  font-size: 20px;
+}
+
+.white {
+  color: #fff !important;
+}
```

Comment:
- [ ] **[OPTIONAL]** Kindly, I would suggest keeping convert this file to CSS and keeping only one file for App styling. This will greatly help to make the project accessible and understandable for all team members 👍 😃